--- hhvm-HHVM-2.3.2/hphp/CMakeLists.txt
+++ hhvm-HHVM-2.3.2/hphp/CMakeLists.txt
@@ -27,12 +27,9 @@
 
 add_subdirectory(third_party/libafdt)
 add_subdirectory(third_party/libmbfl)
-add_subdirectory(third_party/libsqlite3)
 add_subdirectory(third_party/timelib)
-add_subdirectory(third_party/lz4)
 add_subdirectory(third_party/double-conversion)
 add_subdirectory(third_party/folly)
-add_subdirectory(third_party/libzip)
 if(ENABLE_FASTCGI)
 	add_subdirectory(third_party/ti)
 	add_subdirectory(third_party/thrift)
